master/getting-started/docker/tutorials/basic.md

```diff
@@ -72,13 +71,7 @@ You should see output like this on each node
 This worked example creates three Docker networks, where containers on a
 particular network are isolated from containers in the other networks.
 
-### 3.1 Select the IPAM driver
-
-Before we create the networks, we need to select which IP address management
-(IPAM) driver we will use.  There are two options, where we suggest using the
-Calico IPAM driver where possible.
-
-#### The Calico IPAM driver
+### 3.1 The Calico IPAM driver
```

Comment:
not sure I understand, I removed " Select the IPAM driver" section, so had to make it a subsection.